NEWS.md

```diff
@@ -4,6 +4,15 @@ If you read this from a place other than <https://mc-stan.org/projpred/news/inde
 
 # projpred 2.5.0.9000
 
+## Major changes
+
+* Until now, `solution_terms.vsel()` was the only function for extracting a solution path from `vsel` objects (returned by `varsel()` or `cv_varsel()`), namely the solution path from the full-data search. This function is unchanged by this release, to avoid breaking changes for users. However, in anticipation of a larger overhaul of the **projpred** user interface, this release introduces some new functions that are useful for accessing and investigating solution paths (which are now termed *predictor rankings* by these new functions, a term that is hopefully easier to grasp for new users):
+
+    + Added a new function called `ranking()` which returns the predictor ranking from the full-data search and possibly also the predictor rankings from fold-wise searches in case of cross-validation (CV). (More precisely, `ranking()` is a generic. The only method is `ranking.vsel()`. The output is of class `ranking`.)
+    + Added a new function called `props()` which computes ranking proportions (across CV folds, see `?props` for details) from fold-wise predictor rankings. (More precisely, `props()` is a generic. The main method is `props.ranking()`, but as a shortcut, `props.vsel()` has also been added. The output is of class `props`.)
```

Comment:
As `prop` is a common noun, I think it would be better to not shorten and use `proportions`

Reply:
That's true, but see my answer [below](https://github.com/stan-dev/projpred/pull/406#issuecomment-1527009062).

Reply:
We now agreed to rename `props()` to `cv_proportions()` in an upcoming PR.